lib/identify.c

```diff
@@ -851,8 +851,8 @@ static void rotate_capstone(struct quirc_capstone *cap,
 {
 	struct quirc_point copy[4];
 	int j;
-	int best;
-	int best_score;
+	int best = 0;
+	int best_score = INT32_MAX;
```

Comment:
This is assuming `sizeof(int) == 4` which is not guaranteed right? I believe it should be `INT_MAX` instead.